Unknown property 'ANDROID_SUPPORT_ANNOTATIONS_VERSION

When using the plugin version 2.3.1 I get the following error in the platforms\android\app\build.gradle file
```
Could not get unknown property 'ANDROID_SUPPORT_ANNOTATIONS_VERSION' for object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.
```

The ionic environment is
```

Ionic:

   ionic (Ionic CLI)  : 4.4.0 (C:\Users\XXX\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework    : ionic-angular 3.3.0
   @ionic/app-scripts : 1.3.12

Cordova:

   cordova (Cordova CLI) : 6.5.0
   Cordova Platforms     : android 7.1.4
   Cordova Plugins       : 
com.amitsinha.backashome 0.1.0 "Back As Home"
cordova-android-support-gradle-release 2.0.1 "cordova-android-support-gradle-release"
cordova-photo-library 1.1.4 "PhotoLibrary"
cordova-plugin-advanced-http 1.10.2 "Advanced HTTP plugin"
cordova-plugin-advanced-websocket 1.1.4 "Cordova advanced websocket plugin"
cordova-plugin-android-permissions 1.0.0 "Permissions"
cordova-plugin-ble-central 1.1.4 "BLE"
cordova-plugin-camera 2.3.1 "Camera"
cordova-plugin-chrome-apps-common 1.0.7 "Chrome Apps Common Utils"
cordova-plugin-chrome-apps-sockets-udp 1.3.0 "Chrome Apps Sockets UDP API"
cordova-plugin-compat 1.2.0 "Compat"
cordova-plugin-crosswalk-webview 2.4.0 "Crosswalk WebView Engine"
cordova-plugin-device 1.1.5 "Device"
cordova-plugin-device-motion 2.0.1 "Device Motion"
cordova-plugin-enable-multidex 0.1.3 "Enable Multidex"
cordova-plugin-file 4.3.2 "File"
cordova-plugin-file-transfer 1.6.2 "File Transfer"
cordova-plugin-firebase 2.0.5 "Google Firebase Plugin"
cordova-plugin-gallery-refresh 1.0.55 "GalleryRefresh"
cordova-plugin-geolocation 2.4.1 "Geolocation"
cordova-plugin-headercolor 1.0 "HeaderColor"
cordova-plugin-insomnia 4.3.0 "Insomnia (prevent screen sleep)"
cordova-plugin-ionic-webview 2.3.1 "cordova-plugin-ionic-webview"
cordova-plugin-native-logs 1.0.5 "Native Logs"
cordova-plugin-nativestorage 2.2.0 "NativeStorage"
cordova-plugin-network-information 1.3.1 "Network Information"
cordova-plugin-screen-orientation 2.0.0 "Screen Orientation"
cordova-plugin-shake 0.5.4 "Shake Gesture Detection"
cordova-plugin-splashscreen 4.1.0 "Splashscreen"
cordova-plugin-statusbar 2.3.0 "StatusBar"
cordova-plugin-whitelist 1.3.0 "Whitelist"
cordova-save-image-gallery 0.0.26 "saveImageGallery"
cordova.plugins.diagnostic 3.6.5 "Diagnostic"
es6-promise-plugin 4.2.2 "Promise"
ionic-plugin-keyboard 2.2.1 "Keyboard"

System:

   NodeJS : v8.9.4 (C:\Program Files\nodejs\node.exe)
   npm    : 5.6.0
   OS     : Windows 10

```

Things i've tried to fix this:
upgrade android engine to latest
upgrade cordova to latest
upgrade the target sdk to 27
Force the variable in the platform. global and olugin variable/preference

This helped, i did not try that specific version, all the other versions, 7.0.0 and latest, I did a git clean each time, so the platform was re added. Conclusion is that this works only with 7.1.0 for my case.
Next question, perhaps you could help me with that.
Now each time after a successful build, my build script fails
 i run this `"android": "node ./scripts/gitInfo.js && ionic cordova run android", ` using an npm task and that task fails after launch sucess
```
LAUNCH SUCCESS
[ERROR] An error occurred while running cordova run android (exit code 1).

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! XXX@ android: `node ./scripts/gitInfo.js && ionic cordova run android`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the XXX@ android script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\XXX\AppData\Roaming\npm-cache\_logs\2019-01-09T15_11_28_980Z-debug.log
```

This helped, i did not try that specific version, all the other versions, 7.0.0 and latest, I did a git clean each time, so the platform was re added. Conclusion is that this works only with 7.1.0 for my case.
Next question, perhaps you could help me with that.
Now each time after a successful build, my build script fails
 i run this `"android": "node ./scripts/gitInfo.js && ionic cordova run android", ` using an npm task and that task fails after launch sucess
```
LAUNCH SUCCESS
[ERROR] An error occurred while running cordova run android (exit code 1).

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! XXX@ android: `node ./scripts/gitInfo.js && ionic cordova run android`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the XXX@ android script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\XXX\AppData\Roaming\npm-cache\_logs\2019-01-09T15_11_28_980Z-debug.log
```

--- a/README.md
+++ b/README.md
@@ -111,6 +111,7 @@ Whether to use a dark styled keyboard on iOS
 
 ## Plugin Requirements
 
+* **Cordova CLI**: 7.1.0+
 * **iOS**: iOS 11+ and `cordova-ios` 4+
 * **Android**: Android 4.4+ and `cordova-android` 6.4+
 

--- a/package.json
+++ b/package.json
@@ -25,6 +25,11 @@
       "2.0.0": {
         "cordova-android": ">=6.4.0",
         "cordova-ios": ">=4.0.0-dev"
+      },
+      "3.1.0": {
+        "cordova-android": ">=6.4.0",
+        "cordova-ios": ">=4.0.0-dev",
+        "cordova": ">=7.1.0"
       }
     }
   },

--- a/plugin.xml
+++ b/plugin.xml
@@ -27,6 +27,7 @@
     <repo>https://github.com/ionic-team/cordova-plugin-ionic-webview</repo>
 
 	<engines>
+        <engine name="cordova" version=">=7.1.0"/>
         <engine name="cordova-ios" version=">=4.0.0-dev"/>
         <engine name="apple-ios" version=">=11.0"/>
         <engine name="cordova-android" version=">=6.4.0"/>

